package.json

```diff
@@ -426,25 +431,25 @@
     "yauzl": "^2.10.0"
   },
   "devDependencies": {
-    "@babel/cli": "^7.16.0",
-    "@babel/core": "^7.16.0",
-    "@babel/eslint-parser": "^7.16.3",
-    "@babel/eslint-plugin": "^7.14.5",
-    "@babel/generator": "^7.16.0",
-    "@babel/parser": "^7.16.4",
-    "@babel/plugin-proposal-class-properties": "^7.16.0",
-    "@babel/plugin-proposal-export-namespace-from": "^7.16.0",
-    "@babel/plugin-proposal-nullish-coalescing-operator": "^7.16.0",
-    "@babel/plugin-proposal-object-rest-spread": "^7.16.0",
-    "@babel/plugin-proposal-optional-chaining": "^7.16.0",
-    "@babel/plugin-proposal-private-methods": "^7.16.0",
-    "@babel/plugin-transform-runtime": "^7.16.4",
-    "@babel/preset-env": "^7.16.4",
-    "@babel/preset-react": "^7.16.0",
-    "@babel/preset-typescript": "^7.16.0",
-    "@babel/register": "^7.16.0",
-    "@babel/traverse": "^7.16.3",
-    "@babel/types": "^7.16.0",
+    "@babel/cli": "^7.21.0",
+    "@babel/core": "^7.21.0",
+    "@babel/eslint-parser": "^7.19.1",
+    "@babel/eslint-plugin": "^7.19.1",
+    "@babel/generator": "^7.21.1",
+    "@babel/parser": "^7.21.2",
+    "@babel/plugin-proposal-class-properties": "^7.18.6",
+    "@babel/plugin-proposal-export-namespace-from": "^7.18.9",
+    "@babel/plugin-proposal-nullish-coalescing-operator": "^7.18.6",
+    "@babel/plugin-proposal-object-rest-spread": "^7.20.7",
+    "@babel/plugin-proposal-optional-chaining": "^7.21.0",
+    "@babel/plugin-proposal-private-methods": "^7.18.6",
+    "@babel/plugin-transform-runtime": "^7.21.0",
+    "@babel/preset-env": "^7.20.2",
+    "@babel/preset-react": "^7.18.6",
+    "@babel/preset-typescript": "^7.21.0",
+    "@babel/register": "^7.21.0",
+    "@babel/traverse": "^7.21.2",
+    "@babel/types": "^7.21.2",
```

Comment:
Should these versions match `@babel/core`? Its the same on `main` so maybe we address it there first and backport?

Reply:
Maybe you're right - I wonder why it is not in sync on master - @mistic do you have any opinions on syncing the `@babel/*` versions to that of the core?

What I also wonder, the specifiers with `^` should be able to move around within the same major version, in theory they could resolve all to the same version.

Reply:
:+1:  I believe the versions aren't always in sync (e.g. core is 7.22.8 and register is 7.22.5 currently), no-op releases are skipped but future releases will skip numbers to align, if anyone's curious as to why.